scripts/travis/cleanup.js

```diff
@@ -12,33 +12,44 @@ PouchDB.plugin(require('pouchdb-adapter-http'));
 const db = new PouchDB(`${UPLOAD_URL}/${BUILDS_SERVER}`);
 
 const get = () => {
-  console.log(`Getting builds...`);
+  console.log('Getting builds...');
   return db.allDocs({ endkey: END_KEY, limit: MAX_BUILDS_TO_DELETE });
 };
 
 const markDeleted = response => {
-  return response.rows.map(row => {
-    return {
-      _id: row.id,
-      _rev: row.value.rev,
-      _deleted: true
-    };
-  });
+  return response.rows
+    .filter(row => row.id.startsWith('medic:medic:'))
+    .map(row => {
+      return {
+        _id: row.id,
+        _rev: row.value.rev,
+        _deleted: true
```

Comment:
The root cause was the validate_doc_update failing because this doc doesn't have the recommended fields. I've modified the validate_doc_update to not care if you're messing with a release that doesn't have a major version. Not sure if this is the right approach...

Reply:
I don't really understand what you mean here (do you mean "is a branch"?)

Also, did you just update in place or did you also change the repo: https://github.com/medic/medic-builds-repo ?

Reply:
I completely forgot that repo existed  :(

PR created to update validate_doc_update...